### ISSUE TITLE
Styling

### ISSUE THREAD

**Issue by taylorseager** [taylorseager/INDIVIDUAL-ASSESSMENT-vocab-YOU-lary#29]
# User Story
The user should have a nice looking and functioning website to review vocab cards. 

# Acceptance Criteria

- [ ] cards are evenly spaced across page
- [ ] dynamic based on display size
- [ ] hover over buttons changes them for indication
- [ ] nav bar transitions to drop down on smaller screens

# Dependecies
Everything else needs to be completed FIRST!

# Dev Notes
This should be the last thing before the loom video or any stretch goals
